### DISCUSSION TITLE
REQ: Lastscores as hud-element

### DISCUSSION THREAD

**Discussion by blaps** [ezQuake/ezquake-source#517] [Ideas]
As an observer/caster, I'd like to see a list of previously played games on the server as a HUD object glued to my screen.

**Example output:**
> player/team 1 vs. player/team 2 (matchup)
> score1:score2 → map (result)

**Settings**
`hud_lastscores_show` <0:hidden|1:show matchup(s) and result(s)| 2:show result(s) only>
`hud_lastscores_match` <0:show all games| 1:show all games only from latest matchup in history| 2:show all games matching my name and/or team in history>
`hud_lastscores_time` <set timeframe in amount of hours (H) that history will look back
`hud_lastscores_rows` <sets number of rows to display, default 5>
`hud_lastscores_color_win` <colorize text for victories, default white>
`hud_lastscores_color_loss` <colorize text for losses, default white>
`hud_lastscores_format` <customize format, default %score1:%score2 > %map %date%time>
This in addition to the regular hud cvars like, scale, align, pos, place, item_opacity, frame etc.

Q: Could it be tied with `matchtag` so that it only displays latest results from a current matchtag and players?

**Comment by elenacjeth833-web** [comment]
Mit HUD-Befehlen können Sie die Anzeige der Spielergebnisse anpassen. Verwenden Sie `:hidden|1`, um Begegnungen und Ergebnisse oder nur die Ergebnisse anzuzeigen. `hud_lastscores_match` steuert, welche Spiele angezeigt werden: alle, die letzte Begegnung oder die Spiele, die Ihrem Namen/Team entsprechen. `hud_lastscores_time` legt den Zeitraum für die Snatch Casino https://snatchcasinos.de/ Spielhistorie in Stunden fest. `hud_lastscores_rows` passt die Anzahl der Zeilen an (Standard: 5). `hud_lastscores_color_win` und `_loss` färben Siege/Niederlagen ein. `hud_lastscores_format` passt die Anzeige an; Standard ist `%score1:%score2 > %map`.

**Comment by JesseReed123** [comment]
Oui, il est possible que ces paramètres HUD puissent être liés à un « matchtag » pour n’afficher que les résultats les plus récents des joueurs du même matchtag. Le cvar hud_lastscores_match peut être configuré pour montrer uniquement les parties correspondant à votre nom ou équipe, ou uniquement celles du dernier match dans l’historique. En combinant cela https://fr.juliusfrance.com/ avec la personnalisation du format et du temps de recherche (hud_lastscores_time), vous pourriez limiter l’affichage aux résultats pertinents pour le match en cours, rendant le HUD plus clair et ciblé.
